bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WpToTpBottom/WpToTpBottom_conf/Wb_proc_card.dat

```diff
@@ -0,0 +1,20 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+
+import model sm
+
+define p = g u c d s u~ c~ d~ s~
```

Comment:
Hi, you should remove the duplicate import as only the last one is valid.
also, we recommend to remove the proton and jet definitions
https://twiki.cern.ch/twiki/bin/viewauth/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

Reply:
But there are two different model. Should I delete all lepton, proton, and  jet definition?